bigtop_toolchain/manifests/renv.pp

```diff
@@ -49,22 +49,42 @@
         ]
       }
     }
+    /openEuler/: {
+      $pkgs = [
+        "R",
+        "R-devel",
+      ]
+    }
   }
 
-  if ($operatingsystem != 'openEuler') {
-    package { $pkgs:
-      ensure => installed,
-      before => [Exec["install_r_packages"]]
+  if ($operatingsystem == 'openEuler'){
+    if ($architecture == "aarch64") {
+        $url = "https://github.com/jgm/pandoc/releases/download/2.19.2/pandoc-2.19.2-linux-arm64.tar.gz"
+        $pandoctar = "pandoc-2.19.2-linux-arm64.tar.gz"
+    } else{
+        $url = "https://github.com/jgm/pandoc/releases/download/2.19.2/pandoc-2.19.2-linux-amd64.tar.gz"
+        $pandoctar = "pandoc-2.19.2-linux-amd64.tar.gz"
+    }
```

Comment:
Does openEuler offically support PowerPC?
If not, please add comments here and Bigtop would not offer the openEuler packages for ppc64le in the future.
(We would add this notification in the next release annoucement).

Reply:
openEuler currently does not support PowerPC, i will add comments later.